map_data/geographical_region.txt

```diff
@@ -767,4 +768,98 @@ world_innovation_camel = {
 		world_kalimdor_tanaris
 	}
 }
-
+custom_sea_locked = {
+	regions = { 
+		world_eastern_kingdoms_kul_tiras
+		world_eastern_kingdoms_azeroth_stranglethorn
+		world_eastern_kingdoms_lordaeron_baradin_bay
+		world_northrend_howling_fjord
+		world_south_seas
+	}
+}
+custom_middle_kalimdor = {
+	regions = { 
+		world_kalimdor_azshara
+		world_kalimdor_barrens
+		world_kalimdor_durotar
+		world_kalimdor_dustwallow
+		world_kalimdor_mulgore
+		world_kalimdor_stonetalon
+		world_kalimdor_feralas
+		world_kalimdor_desolace
+		world_kalimdor_thousand_needles
+		world_kalimdor_uldum
+		world_kalimdor_silithus
+		world_kalimdor_tanaris
+	}
+}
```

Comment:
I don't think it should be just the entire Middle Kalimdor, but Azshara, Barrens, Durotar, Dustwallow, Mulgore, Tanaris. These are regions that became home to the developing colonies of Kalimdor.

Reply:
I can agree with removing silithus and thousand needles, but stonetalon, feralas, desolace and uldum all look inhabitable.  I don't think colonization has to strictly be the way it happened it the lore.

Stonetalon,desolcace and feralas also give a good expansion route for highborne and they have colonazation unlocked

Reply:
> I can agree with removing Silithus and Thousand Needles, but Stonetalon, Feralas, Desolace and Uldum all look inhabitable. I don't think colonization has to strictly be the way it happened it the lore.

Inhabitable? Yes, inhabitable. These regions weren't colonized because they were inhabitable before.

Reply:
sorry you misunderstood, inhabitable means the are suitable to live in, not I did not mean uninhabitable.  
Stonetalon is just forest
Feralas in Jungle
Desolace looks mostly habitable with spots of bad land, but if Tanaris is included so should Desolace
Uldum looks mostly like Tanaris with a Nile-like strip of land highly suitable for life

Do you mean someone already lives there?
Let me know if I am wrong.

Reply:
> Do you mean someone already lives there?

Yes. They weren't colonized during the gameplay. There were no colonies.

Reply:
Ok excluding Uldum, because thats tol'vir territory.  What about Stonetalon, feralas and desolace, most non-kalimdor high tech countries wont even be able to reach the western side of kalimdor and it would be a good tech for high tech natives of kalimdor.  It fits with the description of the tech just as much as the other lands do imo.

Reply:
The only issue I can see is that compared to the tech in vanilla, I specify a lot more regions to provide this tech, so that might be a good reason to shrink.